blog/_src/posts/2017-06-06-spring-2017-pl-junior-retrospective.md

```diff
@@ -0,0 +1,165 @@
+    Title: Spring 2017 PL Junior Retrospective
+    Date: 2017-06-06T11:38:25
+    Tags: DRAFT, PL Junior, by Ben Chung, by Milo Davis, by Ming-Ho Yee, by Matt Kolosick, by Dustin Jamner, by Artem Pelenitsyn, by Julia Belyakova, by Sam Caldwell
+
+The [PL Junior Seminar][pljunior] is for beginning PhD as well interested
```

Comment:
(also I think the sentences are a bit too long for their own good; and I'm already a very long-winded English writer)